src/components/UserFilterDropdown/index.tsx

```diff
@@ -0,0 +1,123 @@
+import React, {useEffect, useState, ChangeEvent} from 'react';
+import {useDispatch} from 'react-redux';
+import {displayErrorToast} from 'utils/toasts';
+import {AppDispatch, SFC} from 'types';
+import * as S from './Styles';
+import axios from 'axios';
+import {UserReadSerializer} from 'types';
+import {authorizationHeaders} from 'utils/authentication';
+const BASE_URL = `${process.env.REACT_APP_API_URL}/api/users`;
+
+interface User {
+  id: number;
+  username: string;
+  avatar: string;
+}
```

Comment:
Let's not redefine the User interface.
I think we should already have that, which can be re-used